draft-ietf-dnsop-svcb-https.md

```diff
@@ -1557,13 +1560,13 @@ subregistry of the "Domain Name System (DNS) Parameters" registry:
 
 ## HTTPS RRType
 
-This document defines a new DNS RR type, HTTPS, whose value 65 has
+This document defines a new DNS RR type, "HTTPS", whose value 65 has
 been allocated by IANA from the "Resource Record (RR) TYPEs"
 subregistry of the "Domain Name System (DNS) Parameters" registry:
 
 * Type: HTTPS
 * Value: 65
-* Meaning: Service Binding for HTTPS
+* Meaning: Service Binding for "https"
```

Comment:
```suggestion
* Meaning: Service Binding for HTTP
```

Reply:
As before, I prefer the last option since it affects all HTTP requests.

Reply:
Done